kubectl-plugin/test/e2e/kubectl_ray_job_submit_test.go

```diff
@@ -0,0 +1,129 @@
+package e2e
+
+import (
+	"fmt"
+	"os/exec"
+	"path"
+	"regexp"
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+// Directory when running test is kuberay/kubectl-plugin/test/e2e/
+const (
+	rayJobFilePath           = "../../../ray-operator/config/samples/ray-job.interactive-mode.yaml"
+	rayJobNoEnvFilePath      = "../../../ray-operator/config/samples/ray-job.interactive-mode-no-runtime-env.yaml"
+	kubectlRayJobWorkingDir  = "../../../ray-operator/config/samples/kubectl-ray-submit-job/"
+	entrypointSampleFileName = "entrypoint-python-sample.py"
+	runtimeEnvSampleFileName = "runtime-env-sample.yaml"
+)
+
+var _ = Describe("Calling ray plugin `job submit` command on Ray Job", Ordered, func() {
+	It("succeed in submitting RayJob", func() {
+		cmd := exec.Command("kubectl", "ray", "job", "submit", "-f", rayJobFilePath, "--working-dir", kubectlRayJobWorkingDir, "--", "python", entrypointSampleFileName)
+		output, err := cmd.CombinedOutput()
+
+		fmt.Print(string(output))
+		Expect(err).NotTo(HaveOccurred())
+		// Retrieve the Job ID from the output
+		regexExp := regexp.MustCompile(`'([^']*raysubmit[^']*)'`)
+		matches := regexExp.FindStringSubmatch(string(output))
+
+		Expect(len(matches)).To(BeNumerically(">=", 2))
+		cmdOutputJobID := matches[1]
+
+		// Use kubectl to check status of the rayjob
+		// Retrieve Job ID
+		findJobIdCmd := "kubectl describe rayjob rayjob-sample | grep \"Job Id\" -m1"
```

Comment:
Use `kubectl get` with `jsonpath` instead. Same for others in this file.

`kubectl get rayjob rayjob-sample -o jsonpath='{.status.jobId}'`